public/lib/content-service.js

```diff
@@ -40,8 +40,10 @@ angular.module('wfContentService', ['wfHttpSessionService', 'wfVisibilityService
                     })} 
 
                 getTypes() {
-                    return wfPreferencesService.getPreference('featureSwitch').then((isSwitchActive) => {
+                    return wfPreferencesService.getPreference('featureSwitch')
+                    .then((isSwitchActive) => {
                         return isSwitchActive === true ? this.provideStandardAndNewFormats() : this.provideStandardFormats()})
+                    .catch((err) => {return this.provideStandardFormats()})
```

Comment:
Nice fix, but why would it error?

Reply:
From quickly looking at the getPreference() function in `preferences-service.js`, it looks like it's meant to return a Promise.reject() if it doesn't find a preference set for a particular value - so I guess that needed to be handled.